### YamlMime:UniversalReference
api_name: []
items:
- children:
  - azure.mgmt.media.models.odata_error_py3.ODataError
  fullName: azure.mgmt.media.models.odata_error_py3
  langs:
  - python
  module: azure.mgmt.media.models.odata_error_py3
  name: odata_error_py3
  source:
    id: odata_error_py3
    path: azure-mgmt-media\azure\mgmt\media\models\odata_error_py3.py
    remote:
      branch: HEAD
      path: azure-mgmt-media\azure\mgmt\media\models\odata_error_py3.py
      repo: https://apidrop.visualstudio.com/Content%20CI/_git/ReferenceAutomation
    startLine: 0
  type: module
  uid: azure.mgmt.media.models.odata_error_py3
references:
- fullName: azure.mgmt.media.models.odata_error_py3.ODataError
  isExternal: false
  name: ODataError
  parent: azure.mgmt.media.models.odata_error_py3
  uid: azure.mgmt.media.models.odata_error_py3.ODataError
